Dockerfile

```diff
@@ -6,7 +6,7 @@ WORKDIR /opt/app
 RUN mkdir output
 
 # Upgrade pip and setuptools
-RUN pip install -U pip==19.3.1 setuptools==41.6.0
+RUN python -m pip install -U pip==19.3.1 setuptools==41.6.0
```

Comment:
it’s might be better to use python3 rather than python. The latter might refer to python2 in some environments. Doesn’t matter at all for docker, but might make a difference in readme